.github/pull_request_template.md

```diff
@@ -51,6 +51,9 @@ and would like to solicit the team's feedback. Optionally, provide background
 information regarding your questions and concerns. -->
 
 ### Acceptance Validation
+- [ ] This branch has been deployed to the AWS `dev` environment
```

Comment:
Would it help to link to the `BCDA - Deploy - DEV` job in Jenkins?

Reply:
That's a great thought!  I remember that the first few months on the team it was difficult to remember how to get to Jenkins.  From a formatting perspective, in the comment we'll have to have the full link text; take a look at the most recent commit.